docs/content/ProgressBar.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: ProgressBar
+---
+
+
+Use `ProgressBar` to visualize task completion.
+
+## Default example
+
+```jsx live
+<ProgressBar progress={80}/>
+ ```
+
+## System props
+
+ProgressBar components get `COMMON` system props. Read our [System Props](/system-props) doc page for a full list of available props.
+
+## Component props
+
+| Name | Type | Default | Description |
+| :- | :- | :-: | :- |
+| progress | Number | | Used to set the size of the green bar |
+| large | Boolean | | Uses large variation of progres bar |
```

Comment:
Primer css has a `small` variation (https://primer.style/css/components/progress#small-progress). Are we ok not supporting that? Even if we don't want to support it now, should this be a `size` prop where the only valid  values are `"large" | undefined`?

Reply:
Good call! I thought about that first but for thought the `small` variant was the default, but looks like the default is a height of 8px. I just pushed a commit changing the api to `barSize` because `size` is taken by styled-system in the `COMMON` props. I considered just pointing people to the `height` prop already built into the component via styled-system - but I think in this case we want to set guardrails around the "standard" heights we want people to use with this component.

Reply:
theoretically users could still use the `size` or `height` props and use any pixel value they want, but exposing the `barSize` prop at least gives users some suggested heights which I think is okay!